testsuite/MDAnalysisTests/test_units.py

```diff
@@ -103,4 +103,13 @@ def testForce(self):
         yield self._assert_almost_equal_convert, 2.5, 'kJ/(mol*nm)', 'kJ/(mol*A)', 0.25
         yield self._assert_almost_equal_convert, 1, 'kcal/(mol*Angstrom)', 'kJ/(mol*Angstrom)', 4.184
 
+    def testExceptionUnitUnknownUnit(self):
```

Comment:
Should be `test_exception_unit_unknown_unit` to follow the [naming conventions](https://www.python.org/dev/peps/pep-0008/#function-names). `test_unit_unknown` is much shorter and as descriptive.